vagrant/Vagrantfile

```diff
@@ -37,14 +37,21 @@ puts "[INFO] Shared folder: #{vagrant_root}"
 puts "[INFO] Linux Distro: #{distro}"
 puts "[INFO] Container manager: #{container_manager}"
 
+if distro == :clearlinux and Integer(box[:libvirt][distro][:version]) >= 28510
+  File.exists?("/usr/share/qemu/OVMF.fd") ? loader = "/usr/share/qemu/OVMF.fd" : loader = File.join(File.dirname(__FILE__), "OVMF.fd")
+  if not File.exists?(loader)
+    system('curl -O https://download.clearlinux.org/image/OVMF.fd')
```

Comment:
Shouldn't you check if the download succeeds? And does ruby have a built-in way to do HTTPs requests?

Reply:
@ipuustin according to the Ruby's documentation the [system functio](https://ruby-doc.org/docs/ruby-doc-bundle/Manual/man-1.4/function.html#system)n returns false in case of failure so I can modified as well. Otherwise using a Ruby function I have to manage proxy values but it can be an alternative.